.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
-node_js: 6
+node_js: 8
```

Comment:
Fixes an issue with Travis CI not being able to run yarn

Reply:
We can probably bump this to 12 since we don't use anything lower on LD